src/matrixroomhandler.ts

```diff
@@ -194,4 +354,16 @@ export class MatrixRoomHandler {
       remote,
     };
   }
+
+  private bridgeMatrixRoom(channel: Discord.TextChannel, roomId: string) {
+      const remote = new RemoteRoom(`discord_${channel.guild.id}_${channel.id}_bridged`);
+      remote.set("discord_type", "text");
+      remote.set("discord_guild", channel.guild.id);
+      remote.set("discord_channel", channel.id);
+      remote.set("manual_bridge", true);
```

Comment:
I'm not sure I like the name here. What's the official terminology for it in matrix? Plumbing? ``plumbed_room``.